server/utils/s3.js

```diff
@@ -8,9 +8,29 @@ import fetch from 'isomorphic-fetch';
 import bugsnag from 'bugsnag';
 
 const AWS_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY;
+const AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID;
+const AWS_REGION = process.env.AWS_REGION;
```

Comment:
We'll need to add this new one to `.env.sample`, the addition of this environment variable will make upgrades a breaking change for existing installations. Seems like it's required with v4 too?

Reply:
Yes we will need AWS_REGION

Reply:
Yes we will need AWS_REGION